WordPress/src/main/java/org/wordpress/android/ui/PagePostCreationSources.kt

```diff
@@ -0,0 +1,26 @@
+package org.wordpress.android.ui
+
+enum class PagePostCreationSources(val label: String) {
+    // post created from android shortcut long pressing on app icon
+    POST_FROM_SHORTCUT("post-from-short-cut"),
```

Comment:
I think it should be `post-from-shortcut`

Reply:
Correct...sorry for the typo 😊